lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -307,52 +312,47 @@ def search_filter(run):
             filters=f"{w1:1d}{w2:1d}"
 
     except Exception as e:
-        print(f"\n >>> Exception: {e}")
         raise IOError(
             "--> No mongo DB filter information."
             " You must pass the filters by argument: -f [filters]"
-        )
+        ) from e
```

Comment:
typo?

Reply:
What do you mean?

Reply:
why "from e"?

Reply:
https://docs.python.org/3/tutorial/errors.html#exception-chaining